### PR TITLE
Automating Data Workflows with AWS Step Functions: An Example Using Fivetran and dbt

### PR DESCRIPTION
**Pull request by JMiocevich**

### CODE REVIEW

File: _posts/2023-01-02-automating-data-workflow-stepfunctions.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+font: serif
+title: "Automating Data Workflows with AWS Step Functions: An Example Using Fivetran and dbt"
+date: 2023-01-03
+highlight: monokai
+author: Joseph Miocevich
+image:
```
Comment (Hamo69) [inline _posts/2023-01-02-automating-data-workflow-stepfunctions.md:8]: what image were you thinking?

File: _posts/2023-01-02-automating-data-workflow-stepfunctions.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- `syncFivetranConnectors`, which synchronizes a single Fivetran connector
+- `fivetranWebook`, which is triggered by a webhook from Fivetran and starts a new execution of the state machine
+- An AWS HttpApi endpoint for exposing the fivetranWebook function
+- A DynamoDB table for storing [wait task token](https://docs.aws.amazon.com/step-functions/latest/dg/callback-task-sample-sqs.html) for the state machine
+- An Amazon Elastic Container Service (ECS) cluster for running the ECS task
+- An AWS Step Functions state machine for orchestrating the connectors and running the ECS task
+
+# AWS Step Functions in Action
+
+AWS Step Functions can be used to build a workflow that automates tasks in your system. In this particular solution, a state machine is defined in the statemachine.yml file and outlines the steps in the workflow. The state machine starts by calling a Lambda function called getConnectorList, which retrieves a list of connectors from  Fivetran and stores the result in the state machine's output.
```
Comment (Hamo69) [inline _posts/2023-01-02-automating-data-workflow-stepfunctions.md:42]: is there an example of the getConnectorList output? Just feels a bit light..

**Review by Hamo69 — COMMENTED** [review]
nice - short and sweet. Could be fleashed out some more with examples, but I like it

File: _posts/2023-01-02-automating-data-workflow-stepfunctions.md (hunk 2 above)
Comment (JMiocevich) [reply _posts/2023-01-02-automating-data-workflow-stepfunctions.md]: sure

File: _posts/2023-01-02-automating-data-workflow-stepfunctions.md (hunk 1 above)
Comment (JMiocevich) [reply _posts/2023-01-02-automating-data-workflow-stepfunctions.md]: nothing

**Review by Hamo69 — APPROVED** [review]
LGTM

**Pull request merged**